user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,461 @@
+
+
+
+.. _admin-capabilities-acc:
+
+Abandoned Cart Campaign manipulations
+-------------------------------------
+
+If this capability is set to *"System"*, users with the role can use the Abandoned Cart Campaign functionality to 
+generate automatic mailings to customers who have unfinished their purchase in your Magento store in order to convince 
+them to complete the checkout and place an order.  You can find more details about the functionality in the 
+:ref:`Magento Abandoned Cart Campaigns guide <user-guide-acc>`.
+
+This capability is useful for marketing representatives and other employees working with email campaigns for customers 
+of Magento-based stores, e.g. for a general Magento store manager. The functionality may be reached from the 
+*Marketing* menu:
+
+
+.. important::
+
+    To use the functionality, you also need to :ref:`set up the Abandoned Cart extension <user-guide-acc-ext>`.
+
+    
+.. _admin-capabilities-data-audit:    
+    
+Data Audit
+----------
+
+If this capability is set to *"System"*, users with the role can see full history of changes made to any record of an 
+:ref:`auditable entity <user-guide-entity-management-create-other>` and an out-of-the-box report of all such actions.
+
+
+The report is available from the "*System*" menu and described in more details in the 
+:ref:`Data Audit <user-guide-data-audit>` guide.
+
+The link to the history of changes of a specific record is available in the top right corner of the record’s
+:ref:`View page <user-guide-ui-components-view-pages>`.
+
+|
+
+.. image:: ./img/data_management/view/view_history.png
+
+|
+
+This capability is usually used by system administrators and may be used by managers in order to see how and by whom 
+details of different records were changed.
+
+
+.. _admin-capabilities-dotmailer-stats:    
+
+Dotmailer Statistic
+-------------------
+
+The capability is meaningful if the system has been integrated with dormatiler. 
+
+Such integration enables users to utilize dotmailer as an email campaign engine, as described in the guide on
+:ref:`Integration with Dotmailer <user-guide-dm-integration>`. 
+
+When using the functionality, users with the role can synchronize lists of contacts in OroCRM and dotmailer, and use 
+them to create email campaigns in dotmailer and import them to OroCRM – the dotmailer campaign statistics can then be 
+used to analyze the campaign efficiency. These statistics will only be visible if the capability has been set to 
+*"System"*.
+
+This capability is usually used by marketing managers and representatives responsible for email campaigns.
+
+.. _admin-capabilities-export-entities:    
+
+Export Entities
+---------------
+
+If this capability is set to *"System"*, users with the role can export records of an entity into the a .CSV file, as 
+described in the 
+:ref:`Import and Export Data <user-guide-export>` guide. The :guilabel:`Export` button will 
+appear in the top right corner of the page.
```

Comment:
Edit: If this capability is set to *"System,"* users with the role can export an entity's records into a .CSV file, as described in the :ref:`Import and Export Data <user-guide-export>` guide. The :guilabel:`Export` button will appear in the top right corner of the page.